fe/fe-core/src/main/java/org/apache/doris/planner/SetOperationNode.java

```diff
@@ -185,6 +185,9 @@ public void finalize(Analyzer analyzer) throws UserException {
         // corresponding output slot isn't being materialized)
         materializedResultExprLists.clear();
         Preconditions.checkState(resultExprLists.size() == children.size());
+        if (analyzer.getDescTbl().getTupleDesc(tupleId) == null) {
```

Comment:
what's the effect of this?

Reply:
I met a npe when I make unitest in 
`List<SlotDescriptor> slots = analyzer.getDescTbl().getTupleDesc(tupleId).getSlots();`
because analyzer.getDescTbl().getTupleDesc(tupleId) return null...
 so I add this, maybe it is not important , and it should not fix in this pr. I will delete this.